src/microbe_stage/gui/EditorEntryButton.cs

```diff
@@ -0,0 +1,43 @@
+using Godot;
+
+/// <summary>
+///   Button for entering editor in creature stages
+/// </summary>
+public partial class EditorEntryButton : TextureButton
+{
+    [Export]
+    private TextureRect highlight = null!;
+
+    [Export]
+    private AnimationPlayer buttonAnimationPlayer = null!;
+
+    protected override void Dispose(bool disposing)
+    {
+        if (disposing)
+        {
+            if (highlight != null)
+            {
+                highlight.Dispose();
```

Comment:
These should not be disposed, these aren't NodePaths.

Reply:
My IDE told me these should be.

Reply:
Follow our styleguide and add a pragma warning suppression around the two variables.